src/govuk/i18n.mjs

```diff
@@ -37,12 +37,23 @@ I18n.prototype.t = function (lookupKey, options) {
     // Get the plural suffix
     var pluralSuffix = this.getPluralSuffix(options.count)
 
-    // Overwrite our existing lookupKey
-    lookupKey = lookupKey + '.' + pluralSuffix
+    // Use the correct plural form if provided
+    if (lookupKey + '.' + pluralSuffix in this.translations) {
+      lookupKey = lookupKey + '.' + pluralSuffix
+    // Fall back to `other` if the plural form is missing, but log a warning
+    // to the console
+    } else if (lookupKey + '.other' in this.translations) {
+      if (console && 'warn' in console) {
```

Comment:
With the Developer Tools open does IE8 have `console.warn()` or only `console.log()`?

Sorry can't remember!

Reply:
Unsure – what's the thinking? I'm personally not sure it really matters if the warnings don't appear in IE8.

Reply:
Yep this! Just checking if it matters